test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -683,3 +683,24 @@ Describe "Pwsh exe resources tests" -Tag CI {
         $psversiontable.os | Should -MatchExactly "$($osversion.Major).$($osversion.Minor)"
     }
 }
+
+Describe 'Pwsh startup in directories that contain wild cards' -Tag CI {
+    BeforeAll {
+        $powershell = Join-Path -Path $PsHome -ChildPath "pwsh"
+        $dirnames = "[T]est","[Test","T][est","Test"
+        foreach ( $d in $dirnames ) { new-item -type Directory -path "${TESTDRIVE}/$d" }
```

Comment:
Please suppress output `new-item -type Directory -path "${TESTDRIVE}/$d" > $null`.
Please use common formatting style:
```powershell
 foreach ( $d in $dirnames ) {
    new-item -type Directory -path "${TESTDRIVE}/$d" > $null
}
```

Reply:
sure, although this is slower than:
`$null = New-Item -type Directory -path "${TESTDRIVE}/$d"`